docs/Reference/API-Methods.md

```diff
@@ -3635,6 +3639,7 @@ there is no receipt
                 "cumulativeGasUsed": "0x5208",
                 "from": "0x627306090abab3a6e1400e9345bc60c78a8bef57",
                 "gasUsed": "0x5208",
+                "effectiveGasPrice": "0x1",
```

Comment:
(You don't need to redo the changes you made in other open PRs in the same repo. Rather, all the changes will be merged into the upstream as the PRs get merged. If they conflict with each other, you can fix merge conflicts as they arise.)